src/app/shared/dialogs/dialogs-form.component.html

```diff
@@ -10,9 +10,9 @@ <h1 mat-dialog-title><span><mat-icon>create</mat-icon>{{title}}</span></h1>
 
       <!-- selectbox -->
       <mat-form-field *ngIf="field.type === 'selectbox'" class="full-width">
-        <select mat-select *ngFor="let options of field.options">
-          <option value="{{option.value}}">{{option.text}}</option>
-        </select>
+        <mat-select placeholder='{{field.placeholder}}' formControlName="{{field.name}}" required="{{field.required}}">
+          <mat-option *ngFor ="let option of field.option" [value]="option">{{option}}</mat-option>
```

Comment:
assumption here is that value and label both will be same which might not be the case